docs/source/rllabec2.rst

```diff
@@ -0,0 +1,60 @@
+Running rllab experiments on EC2
+================================
+
+This page covers the process of running rllab experients on an Amazon
+EC2 instance. This tutorial assumes rllab has been installed correctly
+(`instructions <https://rllab.readthedocs.io/en/latest/user/installation.html>`_).
+
+Setting up rllab with AWS
+-------------------------
+
+First, follow the `rllab cluster setup
+instructions <https://rllab.readthedocs.io/en/latest/user/cluster.html>`__
+with region ``us-west-1``. Modify ``rllab/config_personal.py`` to
+reference the most current Flow Docker image.
```

Comment:
can we add the name of the most current Flow docker image? @kjang96 which one are you using?

Reply:
@fywu85 @kjang96 @lucfisc any updates on the docker image to reference?

Reply:
fywu85/flow

Reply:
@fywu85 @lucfisc @kjang96 we should figure out what docker image to mention here